scripts/generate_test_keys.sh

```diff
@@ -8,20 +8,27 @@
 set -e
 
 
-echo -e  "[SAN]\nsubjectAltName=DNS:*.example.com,DNS:localhost" >> /etc/ssl/openssl.cnf
-
 echo "Generating sidetree-mock Test PKI"
-cd /opt/go/src/github.com/trustbloc/sidetree-mock
+
+cd /opt/workspace/sidetree-mock
```

Comment:
you should re-use the sandbox CA script rather than creating a new one.

Reply:
See https://github.com/trustbloc/edge-sandbox/blob/master/scripts/generate_test_keys.sh#L34

Reply:
this is just for bdd-test

Reply:
and https://github.com/trustbloc/edge-sandbox/blob/master/Makefile#L120

Reply:
I add TODO